### DISCUSSION TITLE
featured image in file parameter

### DISCUSSION THREAD

**Discussion by newbeelearn** [nunocoracao/blowfish#724] [Q&A]
I see that to get feature image associated with the article we need to create a new folder and add feature image there. Is it possible to specify featured image as parameter in md file and place images in general location where we keep all the images?

**Comment by nunocoracao** [comment]
Yes, I think so. Please check Hugo's documentation. The engine supports such use-cases.

**Reply by newbeelearn** [reply]
it looks like there is a bug. When i specify images in my article parameter theme populates meta tags correctly but does not show featured images. should we move this to issues?

**Comment by newbeelearn** [comment]
in simple.html adding this works under .params.image `"<div` class="{{ $articleImageClasses }}" style="background-image:url({{ . | absURL }});"></div>" 
however i am not sure if this is the right way.
